speccy/machine.go

```diff
@@ -49,7 +48,15 @@ func (m *Machine) Start() error {
 	}
 	m.z.RegisterInputHandler(func(addr uint16) byte { return m.keys.keyboardInputHandler(addr) })
 	every := time.Second / 50
+
 	go func() {
+		runtime.LockOSThread()
```

Comment:
It would be good to have a comment here linking to the SDL docs (https://wiki.libsdl.org/SDL2/SDL_RenderPresent) - particularly as we aren't actually ensuring we are on the main thread. So if we hit that problem in the future, we know why we did this.

Reply:
Added comment explaining thread situation